mungegithub/mungers/milestone-maintainer.go

```diff
@@ -158,7 +158,7 @@ var (
 
 	milestoneStateConfigs = map[milestoneState]milestoneStateConfig{
 		milestoneCurrent: {
-			title: "Milestone Issue **Current**",
+			title: "Milestone Issue: **Current Release**",
```

Comment:
Based on [this](https://github.com/kubernetes/test-infra/pull/5356/files#diff-041899fa1d7e8463cffb163c78b51795R62) `**Current**` means "no changes required" or "up to date", not "current release".
Is that comment correct?

Reply:
Thanks for pointing that out. This meaning is not obvious to a non-native speaker, especially one not intimately familiar with the process.
How about we change it to: "Milestone Issue **Labels up-to-date**", then?
Since `No changes required` might be read as "this issue requires no attention", i.e. nobody needs to work on it, which is quite the opposite.

Reply:
Sure that makes sense to me.
@marun WDYT?

Reply:
@marun ping?

Reply:
@marun ping again?

Reply:
/assign marun

Reply:
Sorry, I didn't see this until now.  

`Labels up-to-date` won't be accurate since the updates required during slush/freeze are not label-based.  Not sure about the wording, but maybe something like `Up-to-date for process`?